### ISSUE TITLE
[Feature Request] Time for starting a new day 

### ISSUE THREAD

**Issue by tommy9771** [Stigmatoz/web-activity-time-tracker#97]
Is it possible to provide users with the ability to customize the time frame for determining a new day? For example, allowing them to classify browsing activity before 3am as part of the previous day, and activity after 3am as the current day. Alternatively, users could opt to count browsing activity after 11pm as the next day instead of midnight.

**Comment by Stigmatoz** [comment]
Yes, I think I can add a configurable reset time during the day, after which the time will be collected for a new day.

**Comment by tommy9771** [comment]
> Yes, I think I can add a configurable reset time during the day, after which the time will be collected for a new day.

Thank you for your attention and efforts.

**Comment by alberduris** [comment]
Is this going to be implemented? Would be much appreciated.
